dumoRecordsHandler/Teacher/Teachers.cpp

```diff
@@ -0,0 +1,29 @@
+//
+// Created by emyrmak on 7/24/2019.
+//
+
+#include "Teachers.h"
+
+const int Teachers::idIndex = 0;
+const int Teachers::nameIndex = 1;
+
+
+void Teachers::addRecord(std::vector<std::string> &data) {
+    records.emplace_back(std::stoi(data[idIndex]), data[nameIndex]);
+}
+
+Teachers::~Teachers() {
+    std::cout << "Count of Teachers:" << records.size() << std::endl;
+}
+
+void Teachers::writeToFile(std::string pathToStore) {
+    std::sort(records.begin(), records.end(), comparator<Teacher>);
```

Comment:
Function writeToFile every time is almost identical. Could you consider to create only one implementation?